homeassistant/components/zamg/sensor.py

```diff
@@ -38,7 +38,7 @@
 )
 from .coordinator import ZamgDataUpdateCoordinator
 
-_DType = Union[type[int], type[float], type[str]]
+_DType: TypeAlias = "type[int] | type[float] | type[str]"
```

Comment:
The corresponding `dtype` property in `ZamgSensorEntityDescription` seems unused.
Maybe the property and type alias could be dropped altogether?

Reply:
👍🏻 Will do so in a followup